rabbitmqclient/cluster_reference.go

```diff
@@ -178,25 +178,28 @@ func readUsernamePassword(secret *corev1.Secret) (string, string, error) {
 	return string(secret.Data["username"]), string(secret.Data["password"]), nil
 }
 
-func managementURI(svc *corev1.Service, tlsEnabled bool, clusterDomain string, pathPrefix string) (string, error) {
+func managementURI(svc *corev1.Service, useTLSForConnection bool, clusterDomain string, pathPrefix string) (string, error) {
```

Comment:
This function currently fails with error "failed to find management from service rmq" if `useTLSForConnection` is false and the rabbit cluster only has TLS listener available. It should succeed and return a `https` scheme URI instead of failing.

Reply:
I left the logic in this function the same, but changed how `useTLSForConnection` is computed in line 102. That way, its value will always be `false` if non-TLS listeners are disabled.